protocols/src/N2NP.nroff

```diff
@@ -159,20 +159,20 @@ Frame
 Device Identification
 .LP
 .in 2
-In order to identify a device from an other one, the N2NP use an
-hashed identifier, on 4 bytes. The original hashed content and the size of
-those fields are not decided yet, so those are subjects to changes.
+In order to identify a device uniquely the N2NP uses a hashed identifier with
+a length of 4 bytes. The original hashed content and the size of those fields
+are not finally decided yet, so those are subjects to changes.
```

Comment:
`finally` is unnecessary.

A `s` needs to be removed here:
```
so those are subjects to changes.
                    ^
```

Reply:
I agree, the `s` needs to be removed, i missed that

While the `finally` is not necessary here to form a correct sentence it does clear up the implied contradiction of having a size for the fields but saying the size hasn't been decided yet (implication of not having a size for them at all), with the finally, it is immediately clear that the previously mentioned sizes are valid for now, but might change at any point in time